CHANGELOG.md

```diff
@@ -1,7 +1,31 @@
 # Upcoming Release
 
 ## New Features:
-No changes to highlight.
+
+### New `gr.BarPlot` component! 📊
+
+Create interactive bar plots from a high-level interface with `gr.BarPlot`. 
+No need to remember matplotlib syntax anymore!
+
+Example usage:
+
+```python
+gr.BarPlot.update(
+    barley,
```

Comment:
Would be good to include the definition of `barley` in this example so people understand that it is supposed to be a dataframe

Reply:
100%